src/main/resources/portal.properties.EXAMPLE

```diff
@@ -231,10 +231,9 @@ oncoprint.hotspots.default=true
 # Custom gene sets
 # querypage.setsofgenes.location=file:/<path>
 
-# to enable use of Ehcache, set ehcache.cache_enabled to true and uncomment all Ehcache properties
-ehcache.cache_enabled=false
+# to enable use of Ehcache, set ehcache.cache_type to a valid cache type (heap, disk, hybrid)
```

Comment:
In the comment, I would add a suffix ... ", to disable use of Ehcache set cache_type to none
That way a user who has enabled the cache will know how to disable it again.
Maybe a simpler comment would be # valid cache types are (heap, disk, hybrid), or use 'none' to disable use of Ehcache

Reply:
Done.